app/views/posts/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1> show all posts </h1>
```

Comment:
- [ ] Also for the view files we need to create the `index` view templates as we do for the `show`. Please create a file `app/views/users/index.html.erb` and `app/views/posts/index.html.erb` files. These files are going to be used with the `index` action of the `user` and `post` controller actions.